apps/webcomponents/src/app/components/gn-search-input/gn-search-input.component.ts

```diff
@@ -19,33 +19,25 @@ import { CatalogRecord } from '@geonetwork-ui/common/domain/model/record'
   encapsulation: ViewEncapsulation.ShadowDom,
   providers: [SearchFacade, SearchService],
 })
-export class GnSearchInputComponent
-  extends BaseComponent
-  implements AfterViewChecked
-{
+export class GnSearchInputComponent extends BaseComponent {
   @Input() openOnSearch: string
   @Input() openOnSelect: string
-  @ViewChild('searchInput') searchInput: FuzzySearchComponent
+  @Input() placeholder = 'Search'
```

Comment:
This won't be translated. I don't think we need to make the value configurable via an input.